src/multiregex/__init__.py

```diff
@@ -36,12 +36,8 @@
 )
 
 import ahocorasick
-import pkg_resources
```

Comment:
`flit` is incompatible with this. There is a package `flit_scm` that would allow taking version from version control, but doesn't work with latest version of `flit` yet.